ray-curator/tests/stages/text/test_filters.py

```diff
@@ -124,7 +124,6 @@ def test_score_filter(self, letter_count_data: DocumentBatch) -> None:
         letter_filter = LetterCountFilter()
         filter_step = ScoreFilter(letter_filter, text_field="documents")
 
-        filter_step.setup()
```

Comment:
Why did we remove this, just curious ?

Reply:
The `ScoreFilter` setup function really does nothing unless it is a FastText filter or the `TokenCountFilter`. But I may rewrite to do what you did for modifiers: have a helper function that calls setup and process instead of writing it in for each test.